FND/Organizations/Organizations.rdf

```diff
@@ -86,9 +85,8 @@
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasAddress"/>
-				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
-				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+				<owl:onProperty rdf:resource="&lcc-lr;hasMember"/>
```

Comment:
Very unclear how one would use hasPart vs hasMember when representing an org structure/ hierarchy. Especially since these properties are soo generic (from LCC)

Reply:
> Very unclear how one would use hasPart vs hasMember when representing an org structure/ hierarchy. Especially since these properties are soo generic (from LCC)

hasPart is transitive, but hasMember is not, among other differences.  But you're right - when we work through the organization membership situational pattern we can create a more specific property locally if needed.